{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "1f78a218_345e15dc",
        "filename": "libusbhost/usbhost.c",
        "patchSetId": 3
      },
      "lineNbr": 735,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2023-10-23T14:43:05Z",
      "side": 1,
      "message": "seems wrong to add a case that returns failure without setting errno.\n\nis there a reason you\u0027re not addressing this where dev gets set to an invalid value instead?\n\nseems to me like usb_device_new() should fail if given a bad fd? otherwise you\u0027re going to spend the rest of your life chasing everywhere someone tries to use fd in an object you should never have allowed to be created in the first place (including as new uses of fd are added).",
      "revId": "6b6fad494d0217582b2fa9cd992c47050eb5210b",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}